packages/firestore/src/core/sync_engine.ts

```diff
@@ -707,8 +707,8 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
     return this.enableNetwork();
   }
 
-  shutdown(): Promise<void> {
-    return this.disableNetwork();
+  async shutdown(): Promise<void> {
+    this.networkAllowed = false;
```

Comment:
But don't we _want_ to close / clean up network connections when the client is shut down?

Reply:
FirestoreClient does this on shutdown: https://github.com/firebase/firebase-js-sdk/blob/f6fd8f52be282cc8e5f2e7a21948262eaffb343c/packages/firestore/src/core/firestore_client.ts#L372

Reply:
Agh, sorry. I missed this and your conversation with Gil about this already...